python/tank/util/shotgun.py

```diff
@@ -1141,9 +1147,79 @@ def _create_published_file(tk, context, path, name, version_number, task, commen
                            created_by_user, created_at, version_entity, sg_fields=None):
     """
     Creates a publish entity in shotgun given some standard fields.
+
+    :param tk: :class:`~sgtk.Sgtk` instance
+    :param context: A :class:`~sgtk.Context` to associate with the publish. This will
+                    populate the task and entity link in Shotgun.
+    :param path: The path to the file or sequence we want to publish. If the
+                 path is a sequence path it will be abstracted so that
+                 any sequence keys are replaced with their default values.
+    :param name: A name, without version number, which helps distinguish
+               this publish from other publishes. This is typically
+               used for grouping inside of Shotgun so that all the
+               versions of the same "file" can be grouped into a cluster.
+               For example, for a maya publish, where we track only
+               the scene name, the name would simply be that: the scene
+               name. For something like a render, it could be the scene
+               name, the name of the AOV and the name of the render layer.
+    :param version_number: The version number of the item we are publishing.
+    :param task: Shotgun Task dictionary to assciate with publish or none
+    :param comment: Comments string to associate with publish
+    :param published_file_type: Shotgun publish type dictionary to
+                associate with publish
+    :param created_by_user: User entity to associate with publish or None
```

Comment:
`None`

Also, maybe specify that "current" is what is returned by `sgtk.util.get_current_user()`. It might be confusing considering we also have sgtk.get_authenticated_user(), which is not the same thing.